repl/repl.go

```diff
@@ -17,30 +17,56 @@ import (
 	"github.com/influxdata/flux"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/internal/spec"
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/libflux/go/libflux"
+	"github.com/influxdata/flux/memory"
 	"github.com/influxdata/flux/semantic"
 	"github.com/influxdata/flux/values"
 )
 
 type REPL struct {
-	scope   values.Scope
-	querier Querier
 	ctx     context.Context
 	deps    flux.Dependencies
+	querier Querier
+
+	scope    values.Scope
+	itrp     *interpreter.Interpreter
+	analyzer *libflux.Analyzer
+	importer interpreter.Importer
 
 	cancelMu   sync.Mutex
 	cancelFunc context.CancelFunc
 }
 
+var prelude = []string{
+	"universe",
+}
```

Comment:
I thought there was another package that was part of the prelude?

Reply:
I was a bit confused about that. I thought that `influxdata/influxdb` was only imported in influxdb. I'll re-add it since I'm not sure and I think it was included by default.